modules/tinymce/src/core/test/ts/browser/undo/UndoKeyboardShortcutsTest.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.undo.UndoKeyboardShortcutTest', () => {
+  const platform = PlatformDetection.detect();
+
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, [], true);
+
+  // idk why but it works, seperate test command for macOS, Firefox, and the rest
+  const selectAll = (editor: Editor) => {
+    editor.execCommand('SelectAll');
+  };
+  const undoCommand = (editor: Editor) => {
+    TinyContentActions.keystroke(editor, 'Z'.charCodeAt(0), platform.os.isMacOS() ? { metaKey: true } : { ctrl: true });
+
+  };
+  const redoCommand = (editor: Editor) => {
+    TinyContentActions.keystroke(editor, 'Y'.charCodeAt(0), platform.os.isMacOS() ? { metaKey: true } : { ctrl: true });
+
+  };
```

Comment:
These names are misleading as they aren't running a command, so please rename them to be more accurate.

You also still have the line spacing issues here as well. It looks like you may have put the new lines in the wrong place, as they should be between the functions not inside them.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/4cb9179c81f47841bf2fedc0e160071fd5f68cb5